src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -124,6 +124,10 @@ def __init__(self, config):
         if config.rotary_dim is not None:
             self.rotary_dim = config.rotary_dim
 
+        sin, cos = fixed_pos_embedding(x=torch.zeros(1, 1, self.rotary_dim), seq_len=max_positions)
```

Comment:
If everything else works, before merge, it would be a good idea to explain why we can set the second `1` in `(1, 1, self.rotary_dim)` in a comment, so we don't need to read the `fixed_pos_embedding` just to figure this out.

Reply:
`self.rotary_dim` could be `None`, and we will have to deal with this case too, otherwise the code will fail in such case.